packages/protocol/contracts/shared/bridge/Bridge.sol

```diff
@@ -495,7 +495,7 @@ contract Bridge is EssentialContract, IBridge {
         returns (bool success_)
     {
         assert(_message.from != address(this));
-
+        if (_message.data.length == 0) return true;
```

Comment:
If the call is a simple ETH transfer, than the message data length would be zero no, and the internal call would still have to be done?

Reply:
True, now making sure the value must also be zero!